new options to fetch in openstack, enable pass arguments to shell, fix for hostmanager enabled (#895)
theforeman/forklift: README.md

    management_network_address: 172.23.99.0/24
```

Example with openstack provider:

> You will need to install the [vagrant openstack provider](https://github.com/ggiamarchi/vagrant-openstack-provider).

Reviewer: A blank line above this would enhance readability.

> You will need to install vagrant openstack provider. For more information click [here](https://github.com/ggiamarchi/vagrant-openstack-provider).

Reviewer 2: I'd format this as

```suggestion
You will need to install the [vagrant openstack provider](https://github.com/ggiamarchi/vagrant-openstack-provider).
```

Do not forget to set openstack API credentials. To use openstack provider as default look [here](https://www.vagrantup.com/docs/providers/default.html).
Tip: You can disable hostmanager by using settings.yaml

> ```

Reviewer: Blank line above this as well please

    management_network_address: 172.23.99.0/24
```

> Example with openstack provider:  

Reviewer 3: Should this have a `####` header?

Author: Other examples doesn't have title.

Reviewer 3: You're right

You will need to install vagrant openstack provider. For more information click [here](https://github.com/ggiamarchi/vagrant-openstack-provider).  
Do not forget to set openstack API credentials.
To use openstack provider as default look [here](https://www.vagrantup.com/docs/providers/default.html).  

> Tip: You can disable hostmanager by using settings.yaml

Reviewer 3: Note that hostmanager is an optional plugin. By default it's not even present. I'd move this line to the section that tells uses it can be installed (https://github.com/theforeman/forklift/blob/04e95ed57dfba7bb2863efa9b064bcfed568d3b0/README.md#poor-mans-dns-aka-etchosts)

> ```

Reviewer 3: You can use yaml after the backticks for syntax highlighting. It'd also be good to describe in which file this is supposed to end up.
